Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2
+
+WORKDIR /app
+
+ENV BASE_URL "https://www.mozilla.org"
+ENV BROWSER_NAME Firefox
+ENV BROWSER_VERSION 39.0
+ENV PLATFORM "Windows 7"
+ENV SAUCE_CREDENTIALS_PATH /tmp/creds
+ENV BUILD_TAG docker
```

Comment:
In Jenkins were we're using this, yes we do set a build_tag based on the jenkins job name https://github.com/mozilla/bedrock/pull/3401/files#diff-ca92073a5860ceb5098b7f64e3cd5cadR11 which is unique. 

You can use the `-e BUILD_TAG=foo` docker run flag to set it in other runs.

Does that sound good enough?